.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - "iojs"
+  - "4"
   - "0.12"
-  - "0.11"
```

Comment:
figure it was never production ready, so want to only support the production ready releases at this point